src/components/InvoicesList/InvoicesList.js

```diff
@@ -125,8 +125,12 @@ const InvoicesList = ({ dependencies: { dopplerBillingApiClient } }) => {
                     {stateInvoices.items.map((invoice, index) => {
                       return (
                         <tr key={index}>
-                          <td>{invoice.accountId}</td>
-                          <td>{invoice.product}</td>
+                          <td>
+                            <FormattedMessage
+                              id={'invoices_list.document_' + invoice.documentType}
```

Comment:
It is arguable, but, personally, I prefer template strings in place of concatenation:

```suggestion
                              id={`invoices_list.document_${invoice.documentType}`}
```

Reply:
Done